pkg/k8shandler/deployment.go

```diff
@@ -110,10 +110,19 @@ func (node *deploymentNode) state() api.ElasticsearchNodeStatus {
 	}
 
 	// check if the configmapHash changed
-	/*newConfigmapHash := getConfigmapDataHash(node.clusterName, node.self.Namespace)
-	if newConfigmapHash != node.configmapHash {
-		rolloutForReload = v1.ConditionTrue
-	}*/
+	/*  This causes strangeness when scaling up nodes
+	newConfigmapHash := getConfigmapDataHash(node.clusterName, node.self.Namespace, node.client)
+	if node.configmapHash == "" {
+		// if we were already scheduled to restart, don't worry? -- just grab
+		// the current hash -- we should have already had our upgradeStatus set if
+		// we required a restart...
+		node.configmapHash = newConfigmapHash
+	} else {
+		if newConfigmapHash != node.configmapHash {
+			rolloutForRedeploy = v1.ConditionTrue
+		}
+	}
+	*/
```

Comment:
nit: can you just delete this code?

Reply:
i'll also remove the other code that would use this functionality